server/src/main/java/org/apache/iotdb/db/engine/compaction/TsFileIdentifier.java

```diff
@@ -166,6 +166,24 @@ public String getFilename() {
     return filename;
   }
 
+  public String getFilePath() {
```

Comment:
Only path in first data dirs will be return. Using `getFileFromDataDirs().getPath()` will be better.

Reply:
But if targetFile does  not exist, getFileFromDataDirs() will return null.